lmdeploy/turbomind/deploy/converter.py

```diff
@@ -197,11 +230,17 @@ def main(model_name: str,
             default to the default cache directory of huggingface.
         kwargs (dict): other params for convert
     """
-
-    assert model_name in MODELS.module_dict.keys(), \
-        f"'{model_name}' is not supported. " \
-        f'The supported models are: {MODELS.module_dict.keys()}'
-
+    if model_name:
+        logger.warning(
+            'The argument `<model_name>` is deprecated and unused now. '
+            'It will be removed on 2024.12.31. It was originally used to '
+            'specify the name of the built-in chat template, but now it '
+            'is substituted with a clearer parameter `--chat-template`')
+    if chat_template is None:
+        chat_template = best_match_model(model_path)
+    assert chat_template in MODELS.module_dict.keys(), \
+        f"chat template '{chat_template}' is not a built-in template. " \
+        f'The built-ins are: {MODELS.module_dict.keys()}'
```

Comment:
If the user doesn't provide chat_template and the best_match_model return base, should we treat this as an error?

Reply:
Let's keep it unchanged in the PR. We can discuss about it internally later on